sql/updates/world/master/2018_02_04_21358.sql

```diff
@@ -3,6 +3,6 @@ SET @ENTRY := 24745;
 UPDATE `creature_template` SET `AIName`="SmartAI", `ScriptName` = "" WHERE `entry`=@ENTRY;
 DELETE FROM `smart_scripts` WHERE `entryorguid`=@ENTRY AND `source_type`=0;
 INSERT INTO `smart_scripts` (`entryorguid`,`source_type`,`id`,`link`,`event_type`,`event_phase_mask`,`event_chance`,`event_flags`,`event_param1`,`event_param2`,`event_param3`,`event_param4`,`action_type`,`action_param1`,`action_param2`,`action_param3`,`action_param4`,`action_param5`,`action_param6`,`target_type`,`target_param1`,`target_param2`,`target_param3`,`target_x`,`target_y`,`target_z`,`target_o`,`comment`) VALUES
-(@ENTRY,0,0,0,6,0,100,0,0,0,0,0,11,44335,2,0,0,0,0,7,0,0,0,0,0,0,0,"Pure Energy - On Just Died - Cast 'Energy Feedback'"),
+(@ENTRY,0,0,0,6,0,100,0,0,0,0,0,75,44335,0,0,0,0,0,7,0,0,0,0,0,0,0,"Pure Energy - On Just Died - Add Aura 'Energy Feedback'"),
```

Comment:
why this change?

Reply:
Originally:
```c++
killer->CastSpell(killer...);
```
However, SMART_ACTION_CAST does not work on SMART_EVENT_DEATH, at least not with this spell.

Reply:
It must work with the cast flag=2

Reply:
It does not work, it's because the target is TARGET_UNIT_CASTER in spell.